k8s/sandbox/common/idam-sprod/idam-web-admin.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-web-admin
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-web-admin
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-web-admin
+    version: 0.2.3
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-admin:preview
+      ingressHost: idam-web-admin-sbox.service.core-compute-sprod.internal
```

Comment:
```suggestion
      ingressHost: idam-web-admin-sprod-sbox.service.core-compute-sandbox.internal
```

Reply:
This requires addition records on the backend application gateway: https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml#L59-L64

Reply:
https://github.com/hmcts/azure-platform-terraform/pull/154